.github/workflows/scripts/benchmarks/export_benchmarks_data.py

```diff
@@ -0,0 +1,89 @@
+"""
+This module includes functionality to convert JSON-XUBM runtime data to a CSV file.
+"""
+import argparse, json, os, csv
+########################################################################
+# Parsing arguments
+########################################################################
+def parse_args():
```

Comment:
For now, not in the workflows because I set the defaults to convenient values.
I use it for local development and it is sensible to keep it for now because of future changes we have in mind.